litex/tools/litex_json2renode.py

```diff
@@ -889,8 +889,9 @@ def generate_resc(csr, number_of_cores, args, flash_binaries={}, tftp_binaries={
 sysbus LoadBinary @{} {}
 """.format(args.bios_binary, hex(opensbi_base))
 
-    for cpu_id in range(0, number_of_cores):
-        result += f"cpu{cpu_id} PC {hex(opensbi_base)}\n"
+    if opensbi_base:
```

Comment:
Could you split this change to a separate commit?

Reply:
done